### ISSUE TITLE
Travis update-generated-swagger-spec.sh failure: curl: (35) error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version

### ISSUE THREAD

**Issue by dcbw** [openshift/origin#11517]
```
===== Verifying API Swagger Spec =====
Generating a fresh spec...
FAILURE: Generation of fresh spec failed:
[WARNING] configure_os_server is deprecated, please use os::start::configure_server
[INFO] Creating certificates for the OpenShift server
Command "create-master-certs" is deprecated, Use 'openshift admin ca' instead.
Generated new key pair as /tmp/openshift/generate-swagger-spec/openshift.local.config/master/serviceaccounts.public.key and /tmp/openshift/generate-swagger-spec/openshift.local.config/master/serviceaccounts.private.key
[INFO] Creating node configuration for the OpenShift server
Generating node credentials ...
Created node config for 172.17.4.30 in /tmp/openshift/generate-swagger-spec//openshift.local.config/node-172.17.4.30
[INFO] Creating boostrap policy files for the OpenShift server
[INFO] Creating master configuration for the OpenShift server
Wrote master config to: /tmp/openshift/generate-swagger-spec/openshift.local.config/master/master-config.yaml
[INFO] To debug: export KUBECONFIG=/tmp/openshift/generate-swagger-spec//openshift.local.config/master/admin.kubeconfig
[WARNING] start_os_master is deprecated, please use os::start::master
[INFO] openshift v1.4.0-alpha.0+88d8070-744
kubernetes v1.4.0+776c994
etcd 3.1.0-alpha.1
[INFO] Server logs will be at:   /tmp/openshift/generate-swagger-spec//logs
[INFO] Config dir is:            /tmp/openshift/generate-swagger-spec//openshift.local.config
[INFO] Using images:             openshift/origin-${component}:latest
[INFO] MasterIP is:              https://127.0.0.1:38443
[INFO] Scan of OpenShift related processes already up via ps -ef    | grep openshift : 
travis    22695  22694  0 05:20 pts/1    00:00:00 /bin/bash /home/travis/gopath/src/github.com/openshift/origin/hack/update-generated-swagger-spec.sh _output/verify-generated-swagger-spec
travis    22979  22695  0 05:20 pts/1    00:00:00 grep openshift
[INFO] Starting OpenShift server
[INFO] OpenShift server start at: 
Mon Oct 24 05:20:36 UTC 2016
/home/travis/gopath/src/github.com/openshift/origin/hack/lib/cmd.sh: line 553: bc: command not found
Running /home/travis/gopath/src/github.com/openshift/origin/hack/lib/start.sh:296: executing 'oc get --raw /healthz --config='/tmp/openshift/generate-swagger-spec//openshift.local.config/master/admin.kubeconfig'' expecting any result and text 'ok'; re-trying every 0.25s until completion or 0.000s...
/home/travis/gopath/src/github.com/openshift/origin/hack/lib/cmd.sh: line 315: bc: command not found
/home/travis/gopath/src/github.com/openshift/origin/hack/lib/cmd.sh: line 315: bc: command not found
/home/travis/gopath/src/github.com/openshift/origin/hack/lib/cmd.sh: line 578: bc: command not found
SUCCESS after 0.000s: /home/travis/gopath/src/github.com/openshift/origin/hack/lib/start.sh:296: executing 'oc get --raw /healthz --config='/tmp/openshift/generate-swagger-spec//openshift.local.config/master/admin.kubeconfig'' expecting any result and text 'ok'; re-trying every 0.25s until completion or 0.000s
/home/travis/gopath/src/github.com/openshift/origin/hack/lib/cmd.sh: line 553: bc: command not found
Running /home/travis/gopath/src/github.com/openshift/origin/hack/lib/start.sh:297: executing 'oc get --raw /healthz/ready --config='/tmp/openshift/generate-swagger-spec//openshift.local.config/master/admin.kubeconfig'' expecting any result and text 'ok'; re-trying every 0.25s until completion or 0.000s...
/home/travis/gopath/src/github.com/openshift/origin/hack/lib/cmd.sh: line 315: bc: command not found
/home/travis/gopath/src/github.com/openshift/origin/hack/lib/cmd.sh: line 315: bc: command not found
/home/travis/gopath/src/github.com/openshift/origin/hack/lib/cmd.sh: line 578: bc: command not found
SUCCESS after 0.000s: /home/travis/gopath/src/github.com/openshift/origin/hack/lib/start.sh:297: executing 'oc get --raw /healthz/ready --config='/tmp/openshift/generate-swagger-spec//openshift.local.config/master/admin.kubeconfig'' expecting any result and text 'ok'; re-trying every 0.25s until completion or 0.000s
[INFO] OpenShift server health checks done at: 
Mon Oct 24 05:20:43 UTC 2016
[INFO] Updating /home/travis/gopath/src/github.com/openshift/origin/_output/verify-generated-swagger-spec/api/swagger-spec:
[INFO] Updating /home/travis/gopath/src/github.com/openshift/origin/_output/verify-generated-swagger-spec/api/swagger-spec/oapi-v1.json from https://127.0.0.1:38443/swaggerapi/oapi/v1...
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (35) error:1407742E:SSL routines:SSL23_GET_SERVER_HELLO:tlsv1 alert protocol version
[ERROR] PID 22695: /home/travis/gopath/src/github.com/openshift/origin/hack/update-generated-swagger-spec.sh:50: `curl -w "\n" "${SWAGGER_API_PATH}${type}/${endpoint}" > "${SWAGGER_SPEC_OUT_DIR}/${type}-${endpoint}.json"` exited with status 35.
[INFO]      Stack Trace: 
[INFO]        1: /home/travis/gopath/src/github.com/openshift/origin/hack/update-generated-swagger-spec.sh:50: `curl -w "\n" "${SWAGGER_API_PATH}${type}/${endpoint}" > "${SWAGGER_SPEC_OUT_DIR}/${type}-${endpoint}.json"`
[INFO]   Exiting with code 35.
[INFO] Dumping etcd contents to /tmp/openshift/generate-swagger-spec//artifacts/etcd_dump.json
[INFO] Tearing down test
[INFO] Pruning etcd data directory...
/home/travis/gopath/src/github.com/openshift/origin/hack/util.sh: line 393: journalctl: command not found
[INFO] Cleanup complete
[FAIL] !!!!! Generate Failed !!!!
```

**Comment by dcbw** [comment]
Possibly caused by https://github.com/openshift/origin/pull/11495 ?

**Comment by liggitt** [comment]
Definitely. Can decide whether to keep TLS 1.2 and try to fix Travis or revert in the morning

**Comment by stevekuznetsov** [comment]
This is fixed with #11571
